dev/build_inplace.py

```diff
@@ -0,0 +1,8 @@
+'''
+This file is used to build the Cython libraries inplace on development systems.
+'''
+
+import os
+
+os.chdir('..')
```

Comment:
Perhaps change this to use the absolute path (with (os.cwd))? You don't know people will cd to the directory first - I'd just run "python dev/build_inplace.py" which wouldn't work as it would refer to the wrong parent.